### PR TITLE
Add direction support to activateBlock, openBlock

### PR DESCRIPTION
**Pull request by IceTank**
Why? Because you cannot open containers if they are at y level 255 because you cannot activate blocks from the top at y level 255. So openContainer does not work at y 255.

### CODE REVIEW

**Comment by u9g** [comment]
failing tests

**Comment by IceTank** [comment]
> failing tests

Jeah and this would break anything still using callbacks. So I think I will put this on hold until callbacks are gone.

**Comment by rom1504** [comment]
blocked by #1482

**Comment by rom1504** [comment]
Can you rebase this on master @IceTank ?

File: lib/plugins/inventory.js [hunk 1]
```diff
@@ -338,8 +338,8 @@ function inject (bot, { version, hideErrors }) {
     }
   }
 
-  async function openBlock (block) {
-    bot.activateBlock(block)
+  async function openBlock (block, direction, cursorPos) {
```
Comment (rom1504) [inline lib/plugins/inventory.js:341]: looks like you forgot the default here

Comment (IceTank) [reply lib/plugins/inventory.js:341]: Yeah I just noticed that there are also types missing

**Comment by IceTank** [comment]
todo: Change direction from number to Vec3

**Comment by u9g** [comment]
has conflicts

**Review by u9g — APPROVED** [review]
LGTM

**Pull request merged**